src/utils.js

```diff
@@ -567,31 +567,79 @@ nv.utils.initSVG = function(svg) {
 Sanitize and provide default for the container height.
 */
 nv.utils.sanitizeHeight = function(height, container) {
-    return (height || parseInt(container.style('height')) || 400);
-};
+    if (height < 0) {
+        debugger;
+    }
+    if (height > 0) {
+        console.log("height = ", height);
+        return height;
+    }
+    else {
+        var ch = parseInt(container.style('height'));
```

Comment:
You need to specify the radix for parseInt, i.e. parseInt(containter.style('height'), 10)

It's also probably a good idea to use isNaN afterward for a sanity check too.